### ISSUE TITLE
Remove reference to "relevant application cache" and replace it with Fetch concepts

### ISSUE THREAD

**Issue by yoavweiss** [w3c/resource-timing#256]
The "relevant application cache" reference no longer leads to anything (the link was removed from HTML).
It would be good to clarify that:
* Resources that are fetched from the memory/preload cache are not included in the timeline (this may be blocking on a definition of the preload/memory cache, so may require some handwaving in the meantime)
* Resources that are fetched from the HTTP cache are included
* Resources that are fetched from Service Workers (e.g. through the Cache API) are included
* Open question: should we care about AppCache? Is it dead yet?

**Comment by nicjansma** [comment]
Agreed with your first three clarifications.

The naming of the phase "AppCache" in the diagram I find confusing, as I don't think it was put there to be exactly the [AppCache](https://developer.mozilla.org/en-US/docs/Web/HTML/Using_the_application_cache) browser feature specifically... I think it was just a naming choice we used to refer to the HTTP cache lookup times.

![image](https://raw.githubusercontent.com/w3c/resource-timing/gh-pages/timestamp-diagram.svg)

The previous version of this diagram called it just `App cache`:

![image](https://user-images.githubusercontent.com/1004649/114448255-4f746480-9ba1-11eb-81cc-c804701bd01f.png)

And when we switched to SVG it was renamed `AppCache`.

So maybe we should just call the phase "Cache Lookup" or something to avoid linking it to the (defunct) browser feature?
